k8s/mongodb-monitoring-agent/mongo-mon-dep.yaml

```diff
@@ -17,22 +17,40 @@ spec:
   replicas: 1
   template:
     metadata:
+      name: mdb-mon-instance-0-dep
       labels:
         app: mdb-mon-instance-0-dep
     spec:
       terminationGracePeriodSeconds: 10
       containers:
       - name: mdb-mon
-        image: bigchaindb/mongodb-monitoring-agent:1.0
+        image: bigchaindb/mongodb-monitoring-agent:2.0
         imagePullPolicy: Always
         env:
-        - name: MMS_API_KEY
-          valueFrom:
-            configMapKeyRef:
-              name: mdb-mon
-              key: api-key
+        - name: MMS_API_KEYFILE_PATH
+          value: /etc/mongod/cloud/api-key
+        - name: CA_CRT_PATH
+          value: /etc/mongod/ssl/ca.pem
+        - name: MONITORING_PEM_PATH
+          value: /etc/mongod/ssl/mdb-mon-instance.pem
```

Comment:
I guess this file path (`/etc/mongod/ssl/mdb-mon-instance.pem`) is hard-coded and shouldn't be changed, but can you confirm?